packages/block-library/src/latest-posts/edit.js

```diff
@@ -56,6 +56,10 @@ class LatestPostsEdit extends Component {
 		);
 	}
 
+	componentWillUnmount() {
+		invoke( this.fetchRequest, [ 'abort' ] );
```

Comment:
Do we support "abort" in apiFetch requests?

Reply:
if we don't, then the hierarchical term selector is wrong too.

Reply:
I don't think we do :). I think we did at some point in apiRequest because we were relying on jQuery but it's not the case anymore AFAIK.

Reply:
Yup, no abort! Ok, I'll do the `this.isStillMounted` thing instead. Can we fix the term selector in another PR? I'm keen to get this in with as few changes as possible so authors get tested asap